### PR TITLE
DDP-4750: Testboston language selector

### PR DESCRIPTION
**Pull request by fat23cat**
- Replaced language selector placeholder in the TestBoston app
- Updated SDK's language selector - component should emit visibility state to control template around the language selector

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts [hunk 1]
```diff
@@ -19,25 +19,25 @@ import { Subscription } from "rxjs";
...
-      <mat-menu #menu="matMenu">
+      <mat-menu #menu="matMenu" class="language-menu">
         <button mat-menu-item *ngFor="let lang of getUnselectedLanguages()" (click)="changeLanguage(lang)">{{lang.displayName}}</button>
       </mat-menu>
     </div>
   `
 })
 export class LanguageSelectorComponent implements OnInit, OnDestroy {
   @Input() isScrolled: boolean;
+  @Output() visibility: EventEmitter<boolean> = new EventEmitter();
```
Comment (charlottedye) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts]: This is smart--I hadn't thought about cases where a second language could be added to a study later on, but this is obviously useful for that case.

**Review by charlottedye — DISMISSED** [review]
Looks good to me!

File: ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts (hunk 1 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts]: Could we rename to `visible` or `isVisible`? Would be more indicative of a boolean value.

Comment (fat23cat) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts]: 6b3113d8d097c846e2613265c5bba53e25c1ff80

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**